packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/observe/TransactionsTable.tsx

```diff
@@ -85,6 +86,14 @@ export const columns = [
       return <TimeAgo timestamp={timestamp} />;
     },
   }),
+  columnHelper.accessor("write", {
+    header: "Timing",
+    cell: (row) => {
+      const write = row.getValue();
+      if (!write) return <span className="text-white/60">N/A</span>;
```

Comment:
would prefer to just leave this blank when not filled in to not populate the hosted explorer with a bunch of un-actionable "N/A"s

and ideally combine this with the time column

Reply:
yeah good idea, updated so now it's combined with the time column